examples/chart/teleport-cluster/values.yaml

```diff
@@ -0,0 +1,47 @@
+##################################################
+# Values that must always be provided by the user.
+##################################################
+
+# clusterName is a unique cluster name,
+# FQDN that does not change, for example
+# teleport.example.com
+clusterName:
+
+##################################################
+# Values that you may need to change.
+##################################################
+
+# Version of teleport image, if different from appVersion in Chart.yaml.
+teleportVersionOverride: ""
+
+# ACME is a protocol for getting certificates
+# acme enables acme protocol
+acme: false
+acmeEmail: ""
+acmeURI: ""
+
+# Set enterprise to true to use enterprise image
+enterprise: false
+
+# If true, create & use Pod Security Policy resources
+# https://kubernetes.io/docs/concepts/policy/pod-security-policy/
+podSecurityPolicy:
+  enabled: true
+
+# Set customConfig to true to use a custom config. Create
+# a config resource, name it as release name and place it the chart namespace.
+customConfig: false
+
+# Labels is a map of key values pairs about this cluster
+labels: {}
+
+##################################################
+# Values that you shouldn't need to change.
+##################################################
+
+# Container image for the agent.
+image: quay.io/gravitational/teleport
+# Enterprise version of the image
+enterpriseImage: quay.io/gravitational/teleport-ent
+# Number of replicas for the agent deployment.
+replicaCount: 1
```

Comment:
nit: maybe add the path to the license file from above here?

Reply:
There is no need to make this configurable for Kubernetes, the less configuration knobs, the better